packages/frontend/components/lobby/announcements.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { ReactElement } from 'react';
+import { connect } from 'react-redux';
+
+import { RootState } from '../../store';
+import { ThemeOptions } from '../../store/userOptions/types';
+
+interface IMessage {
+  id: string;
```

Comment:
Instead of ID, can we have it pass the timestamp? Pls also change the key to use timestamp. See my IMessage interface for how I did it. I think it's better than using ID.

Reply:
But I need a unique key for the React announcement div and timestamp might not be unique sometimes? If you are storing the announcements in the database, there should be an ID anyway

Reply:
Timestamp should always be unique. I guess we can pass over an ID, but I think it's easier to just pass the timestamp and then have that displayed too perhaps. Maybe as a D/M timestamp. E.g. ` 12/2 Morgana cannot fail! `.